src/System.Private.Uri/src/System.Private.Uri.csproj

```diff
@@ -84,17 +84,14 @@
     <Compile Include="$(CommonPath)\Interop\Windows\kernel32\Interop.Normalization.cs">
       <Link>Common\Interop\Windows\kernel32\Interop.Normalization.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\Interop\Windows\normaliz\Interop.Idna.cs">
+    <Compile Include="$(CommonPath)\Interop\Windows\normaliz\Interop.Idna.cs" Condition="'$(TargetGroup)'=='uap101aot'">
       <Link>Common\Interop\Windows\normaliz\Interop.Idna.cs</Link>
-    </Compile>
+  </Compile>
   </ItemGroup>
   <ItemGroup Condition="'$(TargetsUnix)' == 'true'">
     <Compile Include="$(CommonPath)\System\StringNormalizationExtensions.Unix.cs">
       <Link>Common\System\StringNormalizationExtensions.Unix.cs</Link>
```

Comment:
This should be also deleted and just call the normalization implementation in CoreLib.